kolibri/plugins/learn/assets/src/views/LearningActivityBar.vue

```diff
@@ -170,6 +183,12 @@
       };
     },
     computed: {
+      ...mapState({
```

Comment:
I'd suggest not adding dependencies to Vuex in smaller components that don't need it like this learning activity bar. I don't see a reason why not to use  `props` here that are made for that purpose, especially when the amount of information you're adding here is not huge nor complex. This will also simplify the test suite that will otherwise have tendencies to grow complicated even for very simple tests (you won't need `makeStore` and you won't need to know store structure to test a simple UI piece)

Reply:
Another good reason to avoid is that we're starting using Composition API and it'd be good to keep smaller building block components flexible enough to be able to work well no matter if their more complex parent components are built with Vuex or with Composition API.

Reply:
Done and pushed - thanks for catching that Misha!